### ISSUE TITLE
[BUG] object of type 'TimeoutError' using Crypto

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#2316]
**Describe the bug**
- Getting `object of type 'TimeoutError'` when inputting token pair when using `crypto_com` exchange using **Windows binary installer**

**Steps To Reproduce**
create any strategy ARB, XEMM or PMM
select crypto_com as exchange
input pair ETH-USDT (any)

- errors found on logs
```
2020-09-12 18:32:08,471 - 4600 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: object of type 'TimeoutError' has no len()
Traceback (most recent call last):
  File "hummingbot\core\utils\async_utils.py", line 9, in safe_wrapper
  File "hummingbot\client\command\create_command.py", line 63, in prompt_for_configuration
  File "hummingbot\client\command\create_command.py", line 108, in prompt_a_config
  File "hummingbot\client\config\config_var.py", line 55, in validate
  File "hummingbot\strategy\cross_exchange_market_making\cross_exchange_market_making_config_map.py", line 48, in validate_taker_market_trading_pair
  File "hummingbot\client\config\config_validators.py", line 50, in validate_market_trading_pair
TypeError: object of type 'TimeoutError' has no len()

```
**Screenshots**
![image](https://user-images.githubusercontent.com/54516858/92993710-64631980-f526-11ea-9e36-97217019068a.png)
![image](https://user-images.githubusercontent.com/54516858/92993833-7beed200-f527-11ea-96e3-ddd11653eb68.png)
![image](https://user-images.githubusercontent.com/54516858/92993861-c5d7b800-f527-11ea-9f26-7452861453b6.png)

**Release version**
0.31.0

**Attachments**
[logs_hummingbot.log](https://github.com/CoinAlpha/hummingbot/files/5212576/logs_hummingbot.log)

**Comment by rxlxrxsx** [comment]
Can confirm this doesn't happen anymore on latest (v0.32.0)

![image](https://user-images.githubusercontent.com/64377055/96060966-8b659000-0ec4-11eb-8c0e-a4376aa79596.png)
![image](https://user-images.githubusercontent.com/64377055/96061009-a46e4100-0ec4-11eb-8bd3-87b1aa49b088.png)
![image](https://user-images.githubusercontent.com/64377055/96061035-b8b23e00-0ec4-11eb-8f68-e7b709203e90.png)

**Comment by PtrckM** [comment]
thank you @rxlxrxsx confirmed fix on latest version **0.32.0**
